resteasy-jaxrs/bin/jaxrs20/pom.xml

```diff
@@ -0,0 +1,342 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
I believe this file has been added by mistake and should be deleted (/bin/ directory...)

Reply:
@asoldano, aaargh, where did that come from. Gone.